modules/is/manifests/params.pp

```diff
@@ -35,6 +35,44 @@
 
   $start_script_template = 'bin/wso2server.sh'
 
+  $template_list = [
+    'repository/conf/carbon.xml',
+    'repository/conf/user-mgt.xml',
+    'repository/conf/datasources/master-datasources.xml',
+    'repository/conf/axis2/axis2.xml',
+    'repository/conf/identity/identity.xml',
+  ]
+
+  # carbon.xml configs
+  $ports_offset = 0
+  /*
```

Comment:
Why different format used here for comments?

Reply:
Host information is a multi-line comment. It's easier to read it this way.

Reply:
Copyright header is also a multiline comment and you have used a different format in that. Wouldn't it be better to keep all the multiline comments in the same format?